sorts/sorts_testcases.go

```diff
@@ -70,6 +70,15 @@ func makeRandomSignedSlice(size int) []int {
 }
 
 func getSortedVersion(a []int) []int {
```

Comment:
```suggestion
// getSortedVersion Takes a slice of ints and returns it slice as a sorted (in 
// ascending order) copy of the slice passed as an argument.
func getSortedVersion(a []int) []int {
```

or something along those lines. This function is essentially taking in a slice and sorting it with the in-built sorting algorithm. The in-built algorithms sorts it according to the second argument which is passed as a comparison `lambda` (not sure if its called lambda in Golang - mathematically its a lambda). So we can work with the sorted version of the unsorted random slices that we generated for testing to compare the answer of the algorithms with a properly sorted slice.

Reply:
Thanks for suggestion! Just committed.